### PR TITLE
THORN-2055 Reflect project rename and migration in examples

### PR DESCRIPTION
**Pull request by antoinesd**
- changed occurences in readme and doc
- changed artifact in pom.xml files
- detect and corrected problems with maven plugin (THORN-2082)

### CODE REVIEW

**Comment by antoinesd** [comment]
Ribon examples require the merging of thorntail/thorntail#989

**Comment by mkouba** [comment]
add to whitelist

File: README.adoc [hunk 1]
```diff
@@ -16,12 +16,12 @@ These examples demonstrate utilizing and combining different parts of WildFly to
...
 
-By default, this repository uses `SNAPSHOT` versions of WildFly Swarm.  They are used to test
-particular aspects of the WildFly Swarm project in addition to being useful examples of what
+By default, this repository uses `SNAPSHOT` versions of Thorntail.  They are used to test
+particular aspects of the Thorntail project in addition to being useful examples of what
 you can do and how you can do it.
 
 Since these examples are `SNAPSHOT`-centric, our *snapshot* repository is added to the
-root `pom.xml`.  If you were to build https://github.com/wildfly-swarm/wildfly-swarm[WildFly Swarm]
+root `pom.xml`.  If you were to build https://github.com/wildfly-swarm/wildfly-swarm[Thorntail]
```
Comment (mkouba) [inline README.adoc]: Old repo url.

File: docker/docker-jaxrs/pom.xml [hunk 2]
```diff
@@ -9,25 +9,25 @@
...
+    <version>2.0.0.Final-SNAPSHOT</version>
     <relativePath>../</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>example-docker-jaxrs</artifactId>
 
-  <name>WildFly Swarm Examples: Docker and JAX-RS</name>
-  <description>WildFly Swarm Examples: JAX-RS in Docker container</description>
+  <name>Thorntail Examples: Docker and JAX-RS</name>
```
Comment (mkouba) [inline docker/docker-jaxrs/pom.xml]: `-swarm.jar` is used in `docker-maven-plugin` configuration below. Note that `fabric8-maven-plugin` needs upgrade too - see also https://issues.jboss.org/browse/THORN-2048.

Comment (antoinesd) [reply docker/docker-jaxrs/pom.xml]: Is there a temporary release of the plugin or do we have to wait for the release?

**Comment by mkouba** [comment]
@antoinesd It seems that there are some failures in CI builds. Could you verify whether it's a problem with `fabric8-maven-plugin` or some different issue?

**Comment by antoinesd** [comment]
Ribbon multi-service test is [failing](https://ci.wildfly-swarm.io/job/thorntail-examples-pull-request-linux/123/io.thorntail.examples$example-ribbon-test/) because it needs horntail/thorntail#989 to pass

**Comment by mkouba** [comment]
@antoinesd FYI https://github.com/thorntail/thorntail/pull/989 was merged.

**Comment by mkouba** [comment]
retest this please

**Comment by antoinesd** [comment]
retest this please

**Pull request merged**